x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/services/ingest.ts

```diff
@@ -135,10 +135,16 @@ export const sendGetFleetAgentStatusForPolicy = (
  * Get Endpoint Security Package information
  */
 export const sendGetEndpointSecurityPackage = async (
-  http: HttpStart
+  http: HttpStart,
```

Comment:
@oatkiller @paul-tavares should I extract this function to some common directly a couple directory levels up since the upgrade component and the policy_list pages use it?